wdn/templates_4.0/scripts/plugins/mediaelement/mediaelement-and-player.js

```diff
@@ -2957,10 +3031,23 @@ if (typeof jQuery != 'undefined') {
 
 				loading.show();
 				controls.find('.mejs-time-buffering').show();
+                // Firing the 'canplay' event after a timeout which isn't getting fired on some Android 4.1 devices (https://github.com/johndyer/mediaelement/issues/1305)
+                if (mejs.MediaFeatures.isAndroid | mejs.MediaFeatures.isFirefox) {
```

Comment:
fixed in https://github.com/unl/wdntemplates/commit/2e387e0ccf7a124a25d00212db5e37d2abb50f7b